### DISCUSSION TITLE
Generation of InformationalVersion

### DISCUSSION THREAD

**Discussion by axunonb** [devlooped/nugetizer#287] [Q&A] [answered]
Without using NuGetizer the `InformationalVersion` contains the full commit hash, e.g. `2.0.0+9ac9fe78012ae9d6acf0fabbc2bf454666b3fe9d`.
With the NuGetizer package (0.9.2), the version becomes `2.0.0+9ac9fe780`.
Is it possible to control the length of the hash string, that is appended to version?

**Comment by kzu** [comment, marked as answer]
Set `UseShortSourceRevisionId=false` for the project to get the original loooong sha.
This needs documentation :(

**Reply by axunonb** [reply]
This directive is to be included in the `PropertyGroup` section, right?

**Reply by kzu** [reply]
Yes, that's right. You could provide it via a `Directory.Build.props` so it applies to all projects.
